content/doc/book/system-administration/reverse-proxy-configuration-with-jenkins/reverse-proxy-configuration-nginx.adoc

```diff
@@ -108,6 +108,8 @@ server {
 This assumes that you run Jenkins on port 8080.
 Remember to create the folder /var/log/nginx/jenkins.
 
+Make sure you do not end `proxy_pass` value with slash, as it will cause problems like in this [issue](https://community.jenkins.io/t/reverse-proxy-test-does-not-handle-url-correctly/1294/16).
```

Comment:
```suggestion
Make sure you do not end the `proxy_pass` value with a slash, as it will cause problems like in this [issue](https://community.jenkins.io/t/reverse-proxy-test-does-not-handle-url-correctly/1294/16).
```
A small suggestion to help with grammar.